components/more-less/more-less.js

```diff
@@ -51,11 +51,14 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 	static get styles() {
 		return css`
 			:host {
-				display: block;
+				display: flow-root;
 			}
 
 			.d2l-more-less-content {
-				overflow: hidden;
+				display: flow-root;
```

Comment:
This one is only needed for Safari. Without it the button is pushed lower when overflow is clipped for some reason. Nothing to do with margin/padding.